Gemfile

```diff
@@ -11,15 +13,15 @@ gem "pg", "~> 1"
 gem "plek", "~> 2"
 
 group :development do
-  gem "byebug"
-  gem "govuk-lint"
-  gem "listen"
+  gem "byebug", "~> 10"
+  gem "govuk-lint", "~> 3"
+  gem "listen", "~> 3"
 end
 
 group :test do
-  gem "simplecov", require: false
+  gem "simplecov", "~> 0.16"
```

Comment:
do you mean to strip the require from this?

Reply:
Yes - doesn't seem to affect anything...